app/assets/javascripts/handin.js

```diff
@@ -32,7 +32,9 @@ function dragExit(e) {
   $(".drag-drop-handin").get(0).style = undefined;
 }
 
-function clickDrag() {
+function clickDrag(e) {
+  // Ignore shift and tab
+  if (e.key === 9 || e.key === 16) return;
```

Comment:
With the original changes the submission box didn't work for me (neither clicking nor pressing a key worked). The `e` check should ensure `e` is not undefined before accessing `key`, which appears to be the issue. Also, the key codes you had didn't work for me, but "Shift" and "Tab" did.

```suggestion
  if (e && (e.key === "Shift" || e.key === "Tab")) return;
```

Reply:
Changed the names, I believe the undefined check is no longer needed now that event is correctly passed to the function.